packages/x-tailwindcss/tsconfig.json

```diff
@@ -11,8 +12,8 @@
     "sourceMap": true,
     "inlineSources": true,
     "declaration": true,
-    "declarationDir": "./dist/types",
-    "noEmit": true
+    "skipLibCheck": true,
+    "skipDefaultLibCheck": true
```

Comment:
do we need both skipLibChecks? According to the [docs](https://www.typescriptlang.org/tsconfig#skipDefaultLibCheck) I understand that not.

Reply:
I always forget which one is the one I should use 😅. Good catch